### PR TITLE
docs: add openwhisk CN document

### PR DESCRIPTION
**Pull request by penghuima**
Signed-off-by: penghuima <2992426202@qq.com>

### Description

docs: add openwhisk CN document

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/openwhisk.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+| 名称              | 类型    | 必选项 | 默认值  | 有效值       | 描述                                                         |
+| ----------------- | ------- | ------ | ------- | ------------ | ------------------------------------------------------------ |
+| api_host          | string  | 是     |         |              | OpenWhisk API Host 地址，例如 `https://localhost:3233`。     |
+| ssl_verify        | boolean | 否     | true    |              | 当设置为 `true` 时执行 SSL 验证。                            |
+| service_token     | string  | 是     |         |              | OpenWhisk service token，其格式为 `xxx:xxx` ，用于 API 调用时的身份认证 |
+| namespace         | string  | 是     |         |              | OpenWhisk namespace，例如 `guest`。                          |
+| action            | string  | 是     |         |              | OpenWhisk action，例如 `hello`.                              |
+| result            | boolean | 否     | true    |              | 当设置为 `true` 时，获得 action 元数据（执行函数并获得响应结果）。 |
+| timeout           | integer | 否     | 60000ms | [1,60000]ms  | OpenWhisk action 和 HTTP 调用超时（以毫秒为单位）。          |
```
Comment (hf400159) [inline docs/zh/latest/plugins/openwhisk.md:46]:
```suggestion
| timeout           | integer | 否     | 60000ms | [1,60000]ms  | OpenWhisk action 和 HTTP 调用超时时间（以毫秒为单位）。          |
```

**Review by hf400159 — COMMENTED** [review]
👍 great work.

File: docs/zh/latest/plugins/openwhisk.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+`openwhisk` 插件用于将 [Apache OpenWhisk](https://openwhisk.apache.org) 这款开源的分布式无服务器平台作为动态上游集成至 APISIX。
+
+启用 `openwhisk` 插件后，该插件会终止对已配置 URI 的请求，并代表客户端向 OpenWhisk 的 API Host 端点发起一个新的请求，然后  `openwhisk` 插件会将响应信息返回至客户端。
```
[the remark this answers is not in the document]
Comment (penghuima) [reply docs/zh/latest/plugins/openwhisk.md:34]: 这种空格检查您们是用什么工具检查的，有什么好用的工具推荐吗，我在vscode上安装了 AutoCorrect 插件，但貌似该插件并不能帮我检查出来所有空格错误

File: docs/zh/latest/plugins/openwhisk.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+通过以下命令创建一个路由，并在配置文件中添加 `openwhisk` 插件：
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "uri": "/hello",
+    "plugins": {
+        "openwhisk": {
+            "api_host": "http://localhost:3233",
+            "service_token": "23bc46b1-71f6-4ed5-8c54-816aa4f8c502:123zO3xZCLrMN6v2BKK1dXYFpXlPkccOFqm12CdAsMgRU4VrNZ9lyGVCGuMDGIwP",
```
[the remark this answers is not in the document]
Comment (penghuima) [reply docs/zh/latest/plugins/openwhisk.md:98]: When executing `echo ${service_token}`  the output is empty，Where can I set this string.

File: docs/zh/latest/plugins/openwhisk.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (penghuima) [reply docs/zh/latest/plugins/openwhisk.md:34]: What tools do you use to check this kind of space? Are there any useful tools recommended? I installed the AutoCorrect plug-in on vscode, but it seems that the plug-in can't help me check all space errors

File: docs/zh/latest/plugins/openwhisk.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (hf400159) [reply docs/zh/latest/plugins/openwhisk.md:98]: `${service_token}`  代表一个变量。

File: docs/zh/latest/plugins/openwhisk.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (hf400159) [reply docs/zh/latest/plugins/openwhisk.md:34]: sorry, I currently don't have a tool to detect extra spaces.

File: docs/zh/latest/plugins/openwhisk.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+| keepalive_timeout | integer | 否     | 60000ms | [1000,...]ms | 当连接空闲时，保持该连接处于活动状态的时间（以毫秒为单位）。               |
+| keepalive_pool    | integer | 否     | 5       | [1,...]      | 连接断开之前，可接收的最大请求数。                           |
+
+:::note
+
+`timeout` 字段规定了 OpenWhisk action 的最大执行时间，以及 APISIX 中 HTTP 客户端的请求超时时间。
+
+因为 OpenWhisk action 调用可能会耗费很长时间来拉取容器镜像和启动容器，所以如果 `timeout` 字段值设置太小，可能会导致大量的失败请求。
+
+在 OpenWhisk 中 `timeout` 字段的值设置范围从 1 ms 到 60000 ms，历史使用经验建议大家将 `timeout` 字段的值至少设置为 1000ms。
```
Comment (lunarwhite) [inline docs/zh/latest/plugins/openwhisk.md:57]:
```suggestion
在 OpenWhisk 中 `timeout` 字段的值设置范围从 1 ms 到 60000 ms，建议用户将 `timeout` 字段的值至少设置为 1000ms。
```

File: docs/zh/latest/plugins/openwhisk.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+`openwhisk` 插件用于将开源的分布式无服务器平台 [Apache OpenWhisk](https://openwhisk.apache.org) 作为动态上游集成至 APISIX。
+
+启用 `openwhisk` 插件后，该插件会终止对已配置 URI 的请求，并代表客户端向 OpenWhisk 的 API Host 端点发起一个新的请求，然后 `openwhisk` 插件会将响应信息返回至客户端。
```
Comment (lunarwhite) [inline docs/zh/latest/plugins/openwhisk.md:34]: Refer to the [English version](https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/openwhisk.md): `This Plugin can be configured on a Route and requests will be send to the configured OpenWhish API endpoint as the upstream.`

👍Good translation.

File: docs/zh/latest/plugins/openwhisk.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+在 OpenWhisk 中 `timeout` 字段的值设置范围从 1 ms 到 60000 ms，历史使用经验建议大家将 `timeout` 字段的值至少设置为 1000ms。
+
+:::
+
+## 启用插件
+
+### 搭建 Apache OpenWhisk 测试环境
+
+1. 在使用 `openwhisk` 插件之前，你需要通过以下命令运行 OpenWhisk 独立集群模式。请确保当前环境中已经安装 Docker 软件。
```
Comment (lunarwhite) [inline docs/zh/latest/plugins/openwhisk.md:65]: "standalone mode"-> "独立集群模式" or "单节点模式"? 

Or just "stand-alone 模式"? (as other CN docs show)

**Review by lunarwhite — COMMENTED** [review]
Other LGTM!

**Review by SylviaBABY — APPROVED** [review]

**Review by hf400159 — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**